src/Forms/Shared/Samples/Data/AddFeatures/AddFeatures.xaml.cs

```diff
@@ -36,13 +36,16 @@ public AddFeatures()
             Initialize();
```

Comment:
```suggestion
            _ = Initialize();
```
Make this change to resolve the warning.